File: src/Year2022/Day4/PuzzleTest.php
<?php

namespace AdventOfCode\Year2022\Day4;

use AdventOfCode\TestInterface;

class PuzzleTest implements TestInterface
{
    public function testFirstPart(): iterable
    {
        yield '2' => <<<INPUT
2-4,6-8
2-3,4-5
5-7,7-9
2-8,3-7
6-6,4-6
2-6,4-8
INPUT;
    }
    public function testSecondPart(): iterable
    {
        yield '4' => <<<INPUT
2-4,6-8
2-3,4-5
5-7,7-9
2-8,3-7
6-6,4-6
2-6,4-8
INPUT;
    }
}

File: python/Challenges/2024/Day1/puzzleTest.py
class PuzzleTest:
    def test_first_part():
        yield {'result': 11, 'test_input': """
3   4
4   3
2   5
1   3
3   9
3   3
"""
               }

    def test_second_part():
        yield {'result': 31, 'test_input': """
3   4
4   3
2   5
1   3
3   9
3   3
        """
               }

